server/src/instant/storage/sweeper.clj

```diff
@@ -0,0 +1,69 @@
+(ns instant.storage.sweeper
+  (:require
+   [chime.core :as chime-core]
+   [instant.flags :as flags]
+   [instant.model.app-file-to-sweep :as app-file-to-sweep]
+   [instant.storage.s3 :as instant-s3]
+   [instant.util.s3 :as s3-util]
+   [instant.util.tracer :as tracer]
+   [instant.util.date :as date]
+   [clojure.tools.logging :as log])
+  (:import
+   (java.lang AutoCloseable)
+   (java.time Period)))
+
+(defn handle-sweep! []
+  (tracer/with-span! {:name "storage/sweeper"}
+    (let [files (app-file-to-sweep/get-all)
```

Comment:
We should limit the number of rows we fetch here. We want to keep the jobs small because the instance could shut down at any time. The smaller the unit of work, the less chance it will be interrupted.

 Here's how I would set it up:

1. Run the job every hour instead of once a day
2. Take only 500 rows so that you only have to send one batch delete to AWS (it accepts 1000, so 500 gives us room to double while we're deleting legacy keys)
3. Do a checkout of the row, similar to how we do indexing jobs (https://github.com/instantdb/instant/blob/main/server/src/instant/db/indexing_jobs.clj#L341). This will allow multiple machines to run the job at the same time without stepping on each other.